src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -84,6 +84,7 @@
 import org.apache.cassandra.locator.InetAddressAndPort;
 import org.apache.cassandra.locator.Replica;
 import org.apache.cassandra.locator.SeedProvider;
+import org.apache.cassandra.schema.CompressionParams;
```

Comment:
import in the wrong direction (cyclic referencing)

stick with SSTableCompressionOptions in this package, and let the schema package (CompressionParams) import it and do the necessary transformation.